### PR TITLE
Add entry headline

### PR DESCRIPTION
**Pull request by cain**
screen shot 2018-05-28 at 6 19 30 pm
screen shot 2018-05-29 at 11 13 21 am

This is just a pr I created from https://github.com/paulschreiber/liveblog/pull/17/files Just cleaned up a little and some styling changes that were specific to fivethirtyeight

Code stolen from @paulschreiber @adamsilverstein

### CODE REVIEW

**Comment by cain** [comment]
TODO: need to rebuild app.js

File: src/styles/core/editor/_input.scss [hunk 1]
```diff
@@ -66,3 +66,7 @@
 .liveblog-input-cancel:hover {
   color: darken($color-warning, 10%);
 }
+
+.liveblog-input-headline {
+  margin-bottom: .5rem;
+}
```
Comment (paulschreiber) [inline src/styles/core/editor/_input.scss]: this file is missing a trailing newline

**Comment by paulschreiber** [comment]
We're going to try an put together a cleaner PR with our back-end liveblogging, TinyMCE and headline changes.

**Comment by cain** [comment]
@paulschreiber Sounds good. Any eta on when that pr will be done?

We just needed headline now, hence why I created this.

**Comment by paulschreiber** [comment]
Hopefully @adamsilverstein will have time this week.

**Comment by adamsilverstein** [comment]
@cain - 

Excellent, this includes nearly all of the functionality we added for headlines. 

Here is a screenshot of your branch in action:



I noticed one issue: _the headline field should be cleared after posting an update_. 

I see we added that in a way that is tied into the TinyMCE integration, see https://github.com/paulschreiber/liveblog/commit/d04df9132c6fee003b8fad52dfd9e918fbf7ed4d

I'll try to work on a more decoupled approach that we can include here.

**Comment by adamsilverstein** [comment]
@cain I added the field clearing in https://github.com/Automattic/liveblog/pull/474 which builds off this PR since I can't edit this PR directly

Let me know if you have any questions.

**Comment by cain** [comment]
@adamsilverstein Awesome! Thanks for fixing it up 😄 

Do we need to rebuild again? And shall I close this pr?

**Comment by adamsilverstein** [comment]
> Do we need to rebuild again? And shall I close this pr?

Yes, sorry i usually omit built files out of habit to avoid merge conflicts. i'll add those.

yes, you can close this pr.

**Pull request closed without merging**